xcape/core.py

```diff
@@ -75,14 +76,16 @@ def _cape_dummy(p, t, td, ps, ts, tds, source, ml_depth, adiabat, pinc, type_gri
     return cape, cin, mulev, zmulev
 
 def calc_cape(p, t, td, ps, ts, tds, source='surface', ml_depth=500., adiabat='pseudo-liquid',
-         pinc=1000., method='fortran', vertical_lev='sigma'):
+         pinc=1000., method='fortran', vertical_lev='sigma', pres_lev_pos=1):
     """
     Calculate cape for a set of profiles over the first axis of the arrays.
 
     Parameters
     ----------
     p : array-like
         Pressure in mb.
+        When vertical_lev='model', p.shape = t.shape = (nlev, x, y, ...)
+        When vertical_lev='pressure', p.shape = t.shape[0] = (nlev)
```

Comment:
The options for pressure level calculations are starting to be added. This detail will need to be included below in the reshaping step as well.